ancar/portfolio/src/main/java/com/google/sps/servlets/MapServlet.java

```diff
@@ -0,0 +1,54 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Returns map data as a JSON array, e.g. [{"lat": 38.4404675, "lng": -122.7144313}] */
+@WebServlet("/map-data")
+public class MapServlet extends HttpServlet {
+
+    private class elementMap {
+        double lat, lng;
+        String name, description;
+
+        public elementMap(double lat, double lng, String name, String description) {
+            this.lat = lat;
+            this.lng = lng;
+            this.name = name;
+            this.description = description;
+        }
+    }
+
+    private ArrayList<elementMap> markers;
```

Comment:
When you don't need any methods specific to the ArrayList,  it's better to use only necessary interface when defining a variable: 
```java
List<...> markers; 

```